llvm/utils/repo/docker/llvm-prepo/Dockerfile

```diff
@@ -128,83 +123,105 @@ RUN mkdir -p $musl_install_dir/include/linux
 COPY llvm-prepo/futex.h $musl_install_dir/include/linux/futex.h
 RUN cp /usr/include/linux/version.h $musl_install_dir/include/linux/version.h
 
+
 ########################
-# generate dummy files #
+# Generate dummy files #
 ########################
 
 RUN mkdir ${install_dir}/lib/linux/
-RUN repo-create-ticket --output=${install_dir}/lib/linux/clang_rt.crtbegin-x86_64.o --repo=${stdlib_repo} d3540c32449fa62681dc5c22fbc2a353
-RUN repo-create-ticket --output=${install_dir}/lib/linux/clang_rt.crtend-x86_64.o --repo=${stdlib_repo} d3540c32449fa62681dc5c22fbc2a353
-
-RUN ar d ${install_dir}/lib/linux/libclang_rt.builtins-x86_64.a ${install_dir}/lib/linux/clang_rt.o
+RUN repo-create-ticket                                         \
+  --output=${install_dir}/lib/linux/clang_rt.crtbegin-x86_64.o \
+  --repo=${stdlib_repo}                                        \
+  d3540c32449fa62681dc5c22fbc2a353
+RUN repo-create-ticket                                       \
+  --output=${install_dir}/lib/linux/clang_rt.crtend-x86_64.o \
+  --repo=${stdlib_repo}                                      \
+  d3540c32449fa62681dc5c22fbc2a353
+
+RUN ar d ${install_dir}/lib/linux/libclang_rt.builtins-x86_64.a
```

Comment:
It looks like the the file `${install_dir}/lib/linux/clang_rt.o` is removed from the command line. Is it correct?

Reply:
Yes. This command is trying to create an empty archive file and using the ‘d’ command to do it. Deleting clang_rt.o from an archive that doesn’t exist doesn’t make much sense.

Reply:
Actually, I missed that we are doing something similar on two later lines. I‘ll correct them as well.